### PR TITLE
Parse json file

### PR DESCRIPTION
**Pull request by ceddy4395**

### CODE REVIEW

File: smelly_python/code_smell.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
+from multiprocessing.dummy import Array
+
+
+class Code_Smell(object):
```
Comment (wouterpolet) [inline smelly_python/code_smell.py]: I think the naming convention is camelcase for classnames, so `CodeSmell`.
We should probably put pylint in the project already :sweat_smile:

File: smelly_python/code_smell.py [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+        self.line = data['line']
+        self.column = data['column']
+        self.endLine = data['endLine']
+        self.path = data['path']
+        self.symbol = data['symbol']
+        self.message = data['message']
+        self.message_id = data['message-id']
+
+    def __repr__(self) -> str:
+        return f"{self.type} in {self.module} on line {self.line} at {self.column} with reason: {self.message}"
```
Comment (wouterpolet) [inline smelly_python/code_smell.py]: Let's use single quotes everywhere we can

File: smelly_python/code_smell.py [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+    def __str__(self) -> str:
+        return f"{self.type} in {self.module} on line {self.line} at {self.column} with reason: {self.message}"
```
Comment (wouterpolet) [inline smelly_python/code_smell.py]: Same as above, single quotes

File: smelly_python/code_smell.py [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+
+    @staticmethod
+    def convert_dict(json_content) -> Array:
+        ret = []
+        for i in json_content:
+            ret.append(Code_Smell(i))
```
Comment (wouterpolet) [inline smelly_python/code_smell.py:27]:
```suggestion
        for code_smell_dict in json_content:
            ret.append(Code_Smell(code_smell_dict))
```

**Review by wouterpolet — CHANGES_REQUESTED** [review]
Just some nitpicky styling comments :innocent:

File: smelly_python/command_line.py [hunk 5]
```diff
@@ -1,12 +1,13 @@
 import sys
 import json
+from smelly_python.code_smell import Code_Smell
 
 def main():
-    print(f"Argument count: {len(sys.argv)}")
-    if len(sys.argv) < 2: 
-        print('Insufficient parameters provided')
+    if not(len(sys.argv) == 2):
```
Comment (wouterpolet) [inline smelly_python/command_line.py]:
```suggestion
    if len(sys.argv) != 2: 
```

File: smelly_python/code_smell.py (hunk 4 above)
Comment (wouterpolet) [inline smelly_python/code_smell.py]:
```suggestion
        for code_smell_data in json_content:
            ret.append(Code_Smell(code_smell_data))
```

File: smelly_python/code_smell.py (hunk 2 above)
Comment (ceddy4395) [reply smelly_python/code_smell.py]: ack

File: smelly_python/code_smell.py (hunk 3 above)
Comment (ceddy4395) [reply smelly_python/code_smell.py]: ack

File: smelly_python/command_line.py (hunk 5 above)
Comment (ceddy4395) [reply smelly_python/command_line.py]: ack

File: smelly_python/code_smell.py (hunk 1 above)
Comment (ceddy4395) [reply smelly_python/code_smell.py]: ack

**Review by wouterpolet — APPROVED** [review]
lgtm, pylint errors are fixed in another branch so that doesn't need to pass yet imo.

**Pull request merged**